### DISCUSSION TITLE
how to deploy telegram bot

### DISCUSSION THREAD

**Discussion by XodjayevDev** [Muzaffar-Xadjayev/eng-to-uzb-bot#3] [General]
I built telegram bot and I'm not knowing how to deploy it for always work

**Comment by Muzaffar-Xodjayev** [comment]
> I built telegram bot and I'm not knowing how to deploy it for always work

for this , firstly you should choose some server for deploy and then timeweb.com I suggested that server it is good service

**Comment by XodjayevDev** [comment]
> > I built telegram bot and I'm not knowing how to deploy it for always work
> 
> for this , firstly you should choose some server for deploy and then timeweb.com I suggested that server it is good service

thank you so much
